src/Validation/ScannableURLProvider.php

```diff
@@ -71,8 +71,12 @@ public function get_supportable_templates() {
 				'is_singular',
 			];
 			if ( 'page' === get_option( 'show_on_front' ) ) {
-				$allowed_templates[] = 'is_home';
-				$allowed_templates[] = 'is_front_page';
+				if ( get_option( 'page_for_posts' ) && amp_is_post_supported( get_option( 'page_for_posts' ) ) ) {
+					$allowed_templates[] = 'is_home';
+				}
+				if ( get_option( 'page_on_front' ) && amp_is_post_supported( get_option( 'page_for_posts' ) ) ) {
```

Comment:
Oops. This should have been:

```suggestion
				if ( get_option( 'page_on_front' ) && amp_is_post_supported( get_option( 'page_on_front' ) ) ) {
```